src/traces/histogram/index.js

```diff
@@ -12,7 +12,7 @@
 /**
  * Histogram has its own attribute, defaults and calc steps,
  * but uses bar's plot to display
- * and bar's setPositions for stacking and grouping
+ * and bar's crossTraceCalc (formally known as setPositions) for stacking and grouping
```

Comment:
~formally~ formerly

Reply:
done in -> e2360382c4e13dc22b8cce928ad4e1bfa7c4ad87